administrator/components/com_content/forms/article.xml

```diff
@@ -244,9 +243,9 @@
 
 	</fieldset>
 
-	<fields name="attribs">
+	<fields name="attribs" label="COM_CONTENT_ATTRIBS_FIELDSET_LABEL">
```

Comment:
Di we need a label for this tag? Is this used anywhere?

Reply:
It wasn't that I added this label it was always there and my revert of your changes brought it back. I don't see where it is used either so can leave it on the basis it was always there or I can remove it.

Reply:
TBH I don't care :D Let's not change it.